### DISCUSSION TITLE
Can't get extends to work in structurizr/lite

### DISCUSSION THREAD

**Discussion by dtlhlbs** [structurizr/lite#110] [Q&A] [answered]
Hi,

I am trying to use the workspace `extends` keyword, but my workspaces in subdirectories are not being picked up by structurizr/lite.

E.g.

`workspace.dsl` in root - this loads fine:

```
workspace "MySystem" "This is a C4 model for My System" {
    model {

        mySystem = softwareSystem "My System" "The general installation environment" " My System"  {
                myApp container "My App"
        }
...etc.
```

in subdirectory `containers\workspace.dsl`, this doesn't load at all:

```
workspace extends ../workspace.dsl {

    model {

        !ref myApp {
            componentA = component "RequestHandler" ""
        }
    }

    views {
        component myApp "My App" {
            include *
            autoLayout
        }
    }
}
```

**Comment by simonbrowndotje** [comment, marked as answer]
Perhaps this will help -> https://docs.structurizr.com/lite/usage#workspace-path

**Reply by dtlhlbs** [reply]
Thanks Simon, does structurizr/lite not search recursively for `workspace.dsl` files and link them together via `extends`? If it doesn't, that'd be a nice feature! From the link it looks like you can set another root, and that `workspace.dsl` can use `extends` as a direct reference, but no other `*.dsl` are 'discoverable'?
